spec/views/profiles/show.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe "profiles/show" do
+  it "displays profile information" do
+    profile = double(
```

Comment:
You can do `Profile.new`, which shouldn't be much slower and provides some more guarantees that you're not just 100% making up methods. Or `instance_double(Profile, ...attributes...)`.